Sources/xcprojectlint-package/CheckForWhitespaceSpecifications.swift

```diff
@@ -29,13 +35,8 @@ public func checkForWhiteSpaceSpecifications(_ project: Project, errorReporter:
       group.wrapsLines.map(toGroupError(group.id, "wrapsLines")),
     ].compactMap { $0 }
   }
-
-  let toFileError: (FileReference, String) -> (String) -> String = { fileReference, type in
-    { _ in
-      "\(project.absolutePathToReference(fileReference)):0:\(errorReporter.reportKind.logEntry) File “\(fileReference.title)” (\(fileReference.id)) contains white space specification of '\(type)'.\n"
-    }
-  }
-
+  .sorted()
```

Comment:
Sorting both for testing purposes (to get predictable results) and for UX. It's nice to get same error message.